source/blog/articles/2016-07-06-contentful-berlin.md

```diff
@@ -0,0 +1,98 @@
+
+Actually contentful is a tool for developer : an API with a focus on content.
+
+- Developers are happy because it’s not a [bloated backend](https://en.wikipedia.org/wiki/Software_bloat)
+- CMS addicts are happy because they can edit content on a web platform
+- Business is happy because they have a [clear pricing page](https://www.contentful.com/pricing/)
+
+![contentful office in berlin](/assets/images/blog/articles/2016-07-06-contentful-berlin/contentful-hq-berlin.jpg)
+
+_This is where magic happens_
+
+I talked with la lot of smart people who are involved in their product.
+
+As a Chief Marketing Officer, Chris want multiple experiences for Contentful. What if customer were allowed to have built-in solution ? You push the famous big red button and — Bam! Your new website is online with an astonish theme… Maybe they are working on it ? #revelation #stayTunned
+
+Artas is Product Marketing Manager and one of his issue is about about __automation__. It’s funny because it remembers me one of my favorite quote.
+
+> Whatever you do, make sure you are __automating__, because if you aren’t, all you are doing is making it harder for yourself when you revisit the __problem__, and making it even harder for the next __person__ who will inevitably have to __reproduce__ your __solution__.
+> <cite> Githuber at Puppet conf</cite>
+
+I encounted Eugene — Copywriter & he share me an amazing website on Typography [typesetinthefuture.com](https://typesetinthefuture.com/).
+
+But wait it's lunch time!
+![Guten appetit](/assets/images/blog/articles/2016-07-06-contentful-berlin/team-building-meal.jpg)
+_Homemade lemonade & mushroom salad_
+
+After the lunch, I meet Marko, Claus & Jan (Designer & developer).
+
+We talked about how I code / organise atomic design systems like [patternlab](patternlab). We are all obsessed with performance and we share our best tools about image optimsation :
+
+- [svgo](https://github.com/svg/svgo) : A nodejs-based tool for optimizing SVG vector graphics files.
+
+- [ImageOptim-CLI](https://github.com/JamieMason/ImageOptim-CLI) : Automate the optimisations of images.
+
+At the end of the day, I have a great time with the Kr3w : playing table tennis in the chilling zone.
+
+![Contentful chilling zone](/assets/images/blog/articles/2016-07-06-contentful-berlin/contentful-chilling-zone.jpg)
+_Basement chilling zone!_
+
+Before leave the team they offer me an amazing gift : the __goodies box__! What’s include inside ?
+
+- A color book
+- Rubicube
+- Stickers
+- Tshirt
+- and … ? and … ? and … ? Socks!
+
+![contentful goodies with T-shirt & Socks](/assets/images/blog/articles/2016-07-06-contentful-berlin/contentful-goodies.jpg)<br>
+_Tweet about the socks #contentsocks @contenful_
+
+## What next?
+
+If you like ruby, [middleman](https://middlemanapp.com/) and want to dig on contentful [follow me on twitter](https://twitter.com/_flexbox)! I will soon have a surprise for you :)
```

Comment:
Please also finish by talking about "rides" :wink: 
eg. try to answer to these questions:
- what do you think about visiting new cities both as tourist and tech guy/worker?
- will you try to go on remote working with someone the next time?
- etc.